media-sound/whipper/whipper-0.7.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python CD-DA ripper preferring accuracy over speed (forked from morituri)"
+HOMEPAGE="https://github.com/JoeLametta/whipper"
+SRC_URI="https://github.com/JoeLametta/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	media-libs/libsndfile
+	test? ( dev-python/twisted )
+"
+RDEPEND="
+	app-cdr/cdrdao
+	>=dev-libs/libcdio-paranoia-0.94_p2
+	dev-python/cddb-py[${PYTHON_USEDEP}]
+	dev-python/pycdio[${PYTHON_USEDEP}]
+	dev-python/pygobject:3=[${PYTHON_USEDEP}]
+	dev-python/python-musicbrainz-ngs[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	media-libs/flac
+	media-libs/libsndfile
+	media-libs/mutagen[${PYTHON_USEDEP}]
+	media-sound/sox[flac]
+"
+
+PATCHES=( "${FILESDIR}/${P}-cdparanoia-name-fix.patch" )
+
+src_compile() {
+	distutils-r1_src_compile
+	emake -C src
+}
+
+python_test() {
+	# accurip test totally depends on network access
+	rm "${WORKDIR}/${P}/${PN}/test/test_common_accurip.py" || die
```

Comment:
Please move that to `src_prepare` or `python_prepare_all`. You can't remove the same file twice, and `python_test` will be run multiple times if you ever start supporting multiple versions of Python. Also, use relative paths, please.